sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -2965,4 +2965,32 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
       }
     }
   }
+
+  test("SPARK-16602 Nvl/Coalesce") {
```

Comment:
I thought we should have here because `Nvl` is RuntimeReplaceable. (Or, did I do some misunderstanding again?)